e2e/tests/steps_init_test.go

```diff
@@ -59,12 +86,31 @@ func (s *rootSuite) Before() {
 	s.validator = new(validatorPod)
 	s.clientset = clientSet
 	s.validatorKeys = vkmap
+
+	// ADDPR: use pocketk8s to populate
```

Comment:
What's ADDPR?

Reply:
Just a placeholder so I can later add new PRs for each item. Intentionally using a term that is not in the Makefile to avoid confusing myself later.